Doc/library/importlib.rst

```diff
@@ -1303,8 +1304,8 @@ find and load modules.
 
    (``__loader__``)
 
-   The loader to use for loading.  For namespace packages this should be
-   set to ``None``.
+   The :class:`Loader <importlib.abc.Loader>` that should be used when loading
```

Comment:
Rather than using the `class` role, `term` should be used to link the glossary defintion.
```suggestion
   The :term:`Loader` that should be used when loading
```

Reply:
@aeros Thanks, agreed!